pkg/kerrors/bizerrors.go

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2021 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kerrors
+
+import (
+	"errors"
+	"fmt"
+
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"github.com/cloudwego/kitex/pkg/remote/trans/nphttp2/codes"
+	"github.com/cloudwego/kitex/pkg/remote/trans/nphttp2/status"
+)
+
+/**
+ * BizErrorIface is an interface used to define biz errors.
+ * Kitex provides the implementation of this interface by default.
```

Comment:
Kitex provides the default implementation of this interface and you can also implement it yourself.